src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -555,7 +556,7 @@ private void ProcessPingFrame(FrameHeader frameHeader)
 
             // Send PING ACK
             // Don't wait for completion, which could happen asynchronously.
-            Task ignored = SendPingAckAsync(_incomingBuffer.ActiveMemory.Slice(0, FrameHeader.PingLength));
+            _ = LogExceptionsAsync(SendPingAckAsync(_incomingBuffer.ActiveMemory.Slice(0, FrameHeader.PingLength)));
 
```

Comment:
Irrelevant to this PR: is _incomingBuffer guarded against being modified by multiple threads at the same time? I'm wondering if we can get into some inconsistent state

Reply:
Good catch. I do think there's an issue here. We should copy the ping data out of _incomingBuffer before we call SendPingAckAsync.

Reply:
Filed issue #38788 to track this